node/core/dispute-coordinator/src/scraping/mod.rs

```diff
@@ -137,13 +137,13 @@ impl ChainScraper {
 		&mut self,
 		sender: &mut Sender,
 		update: &ActiveLeavesUpdate,
-	) -> Result<Vec<ScrapedOnChainVotes>>
+	) -> Result<(Vec<ScrapedOnChainVotes>, Vec<CandidateReceipt>)>
```

Comment:
Update the doc comment - the function now returns scraped votes and included events.

Reply:
Also consider making it a struct with proper field names, instead of a tuple.